src/build.props

```diff
@@ -1,54 +1,72 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<!--
-  Projects that use the new VS 2017 project system import this file. Projects that
-  use the old (pre-VS 2017) project system import "build.old-project.props" instead.
-  This file gets the simple name "build.props" rather than being called
-  "build.new-project.props" because almost every project in this repo uses the
-  VS 2017 project system.
-  -->
+<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
>P [](start = 1, length = 1)

This file now contents the merged contents of the former build.props, build.common.props (now deleted), and build.old-project.props (now deleleted).